api/models.py

```diff
@@ -113,3 +113,19 @@ def modify_parent(cls, query_param_dict):
         """Modify query parameter dictionary for parent"""
         query_param_dict['parent'] = ObjectId(query_param_dict['parent'])
         return query_param_dict
+
+    @classmethod
+    def validate_params(cls, params: dict):
+        """Validate Node parameters"""
+        status = params.get('status')
+        if status:
+            if status not in list(status.value for status in StatusValues):
```

Comment:
maybe we could use `and` operator instead of a nested `if`?

Reply:
Yes, also replaced `list( )` with `[ ]` as it affects the execution speed.